tests/wpt/metadata-css/css21_dev/html4/left-offset-position-fixed-001.htm.ini

```diff
@@ -0,0 +1,3 @@
+[left-offset-position-fixed-001.htm]
```

Comment:
This test case was a false positive. Both the test and ref were rendered wrongly. This patch fixed the ref, but the test part need to be fixed by #12824